Include/ceval.h

```diff
@@ -7,20 +7,31 @@ extern "C" {
 
 /* Interface to random parts in ceval.c */
 
+/* Deprecated from Python 3.7: Use PyObject_Call() instead.
+ * But note that these difference between PyObject_Call():
+ *
+ *   - args can't be NULL in PyObject_Call().
+ *   - PyObject_Call() doesn't raise TypeError when args is not a tuple
+ *     or kwargs is not a dictionary.  It uses assert() to check it.
+ */
 PyAPI_FUNC(PyObject *) PyEval_CallObjectWithKeywords(
     PyObject *callable,
     PyObject *args,
-    PyObject *kwargs);
+    PyObject *kwargs) Py_DEPRECATED(3.7);
 
-/* Inline this */
+/* Deprecated from Python 3.7: Use PyObject_CallObject() instead. */
```

Comment:
did you ment `Use PyObject_Call()` it seem like this patch remove `PyObject_CallObject` ... Autocompleter got a bit overzealous ?